pages/_app.tsx

```diff
@@ -3,15 +3,22 @@ import type { AppProps } from 'next/app';
 import Head from 'next/head';
 import { global } from '../styles/global';
 import '../public/fonts/index.css';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { ReactQueryDevtools } from 'react-query/devtools';
 
 export default function App({ Component, pageProps }: AppProps) {
+  const queryClient = new QueryClient();
+
   return (
     <>
-      <Head>
-        <title>뽀모 - 같이 뽀모해요!</title>
-      </Head>
-      <Global styles={global} />
-      <Component {...pageProps} />
+      <QueryClientProvider client={queryClient}>
+        <ReactQueryDevtools initialIsOpen={true} />
```

Comment:
p4

이렇게 하면 더 깔끔할 것 같아요~
```suggestion
        <ReactQueryDevtools initialIsOpen />
```